frontend/src/components/OrderPage.js

```diff
@@ -757,50 +724,13 @@ class OrderPage extends Component {
                           secondary={t('Order taker')}
                         />
                         <ListItemAvatar>
-                          <Tooltip enterTouchDelay={0} title={t(this.state.taker_status)}>
-                            <Badge
-                              variant='dot'
-                              overlap='circular'
-                              badgeContent=''
-                              color={this.statusBadgeColor(this.state.taker_status)}
-                            >
-                              <Badge
-                                overlap='circular'
-                                anchorOrigin={{ horizontal: 'left', vertical: 'bottom' }}
-                                badgeContent={
-                                  <div style={{ position: 'relative', right: '5px', top: '2px' }}>
-                                    {' '}
-                                    {this.state.type ? (
-                                      <SendReceiveIcon
-                                        sx={{ height: '18px', width: '18px' }}
-                                        color='secondary'
-                                      />
-                                    ) : (
-                                      <SendReceiveIcon
-                                        sx={{
-                                          transform: 'scaleX(-1)',
-                                          height: '18px',
```

Comment:
Just a small detail has changed here, not too important. The badge with the arrows that indicates whether the user is selling/buying (arrows go away from the avatar if he sells, or go towards the avatar if he buys) is located on the left size of the avatar when it is the taker avatar in the Order page. Visually it helps to understand where the Sats leave and where they go.
How it looks on this PR:
![onPR](https://user-images.githubusercontent.com/90936742/191734201-51350ae9-59b6-4fe0-b1e5-a14b96c3956f.png)
How it used to look:
![before](https://user-images.githubusercontent.com/90936742/191734633-f9e7fc99-c594-4bd8-b6b2-7e4af5707c5a.png)

Reply:
But on the Book the arrow is on the right, right? I have on mind to try to normalize everything so all avatars looks identical so shape and icons becomes familiar for the user. What do you think?